pkg/commands/version.go

```diff
@@ -33,6 +34,7 @@ func (e *Executor) initVersion() {
 		Use:   "version",
 		Short: "Version",
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			cmd.SetOut(os.Stdout)
```

Comment:
I think you can just replace `cmd.Println` `fmt.Println`

Reply:
Sure thing, will do the change.

Reply:
Changed all cobra based prints to `fmt`.